extras/rya.geoindexing/pom.xml

```diff
@@ -62,6 +79,11 @@
             <groupId>org.apache.rya</groupId>
             <artifactId>rya.prospector</artifactId>
         </dependency>
+        <dependency>
+            <groupId>net.sf.jopt-simple</groupId>
+            <artifactId>jopt-simple</artifactId>
```

Comment:
I didn't dig in to see where this is getting used, but can we swap this dependency for JCommander?  I'd prefer to standardize on a single CLI Parsing framework, and JCommander is used throughout Rya.

Reply:
this can be a follow on task - @jdasch can you document in jira?  @DLotts just fix the rat issue and we can merge this.

Reply:
Ug, I just looked deeper.  it is using Strings class.  It was probably supposed to be from Java standard libs or maybe the google lib.